src/database/queries/entity.js

```diff
@@ -55,4 +57,21 @@ module.exports = class UserEntityData {
 			return error
 		}
 	}
+
+	static async getAllEntities(filters, attributes, page, limit, search) {
```

Comment:
Can we extend and reuse the function - findAllEntities

Reply:
that returns only data,  we need data with count as well.  we can do that

Reply:
we have pagination so it won't return all data,  we can't use it